src/controller/CHIPCluster.cpp

```diff
@@ -44,26 +44,14 @@ void ClusterBase::Dissociate()
     mDevice = nullptr;
 }
 
-CHIP_ERROR ClusterBase::SendCommand(CommandEncoder commandEncoder, uint16_t maxCmdLen, Callback::Callback<> * responseHandler)
+CHIP_ERROR ClusterBase::SendCommand(chip::System::PacketBufferHandle & payload, Callback::Callback<> * responseHandler)
```

Comment:
This should pass by value, not by reference, since the intended semantics are that this function takes ownership of the buffer.